frontend/src/components/page-lifeGroups/topLGSummary.js

```diff
@@ -11,12 +12,19 @@ const TopLGSummary = () => {
       </h1>
 
       <div className="flex flex-col lg:flex-row gap-y-5 lg:pb-25 pb-15 content-image">
-        <div className="lg:order-1 text-center image-container">
-          <StaticImage
-            className="max-w-[36.25rem]"
-            src="../../images/lifeGroupHuddle.png"
-            alt="LG people"
-          />
+        <div
+          className={`${mediaWrapper} lg:order-1 text-center image-container`}
+        >
+          <div>
+            <iframe
+              width="560"
+              height="315"
+              src="https://www.youtube-nocookie.com/embed/5OOb0W8Kv_A"
+              title="YouTube video player"
```

Comment:
Should we have a more descriptive title here? Something like "LIFE Group Intro Video